src/core/config/descriptions.h

```diff
@@ -109,4 +109,31 @@ constexpr auto kDInsertStatements = "Rows to be inserted into the table using th
 constexpr auto kDDeleteStatements = "Rows to be deleted from the table using the delete operation";
 constexpr auto kDUpdateStatements = "Rows to be replaced in the table using the update operation";
 constexpr auto kDNDWeight = "Weight of ND to verify (positive integer)";
+constexpr auto kDMinCard =
+        " (1-val) * |R| (where |R| is amount of rows in table R) denotes minimum "
+        "cardinality of a column to be considered a soft key. Value lies in (0, 1)";
```

Comment:
If it includes endpoints wouldn't it be better to write [0, 1]?

Reply:
It doesn't include endpoints